core/src/main/java/org/bouncycastle/crypto/params/Blake3Parameters.java

```diff
@@ -0,0 +1,84 @@
+package org.bouncycastle.crypto.params;
+
+import org.bouncycastle.crypto.CipherParameters;
+import org.bouncycastle.util.Arrays;
+
+/**
+ * Blake3 Parameters.
+ */
+public class Blake3Parameters
+        implements CipherParameters
+{
+    /**
+     * The key length.
+     */
+    private static final int KEYLEN = 32;
+
+    /**
+     * The key.
+     */
+    private byte[] theKey;
+
+    /**
+     * The context.
+     */
+    private byte[] theContext;
+
+    /**
+     * Create a key parameter.
+     * @param pContext the context
+     * @return the parameter
+     */
+    public static Blake3Parameters context(final byte[] pContext)
```

Comment:
It may be worth adding another factory method using a `String` parameter which encodes to UTF-8 as Blake3 encourages use of human-readable unique strings for the KDF context parameter (e.g., an ISO-8601 timestamp and application name).

Reply:
Thanks, I'll note this for when it's added to the JCA provider (the next step).